src/components/visualizations/chart/events/setupDrilldownToParentAttribute.ts

```diff
@@ -11,15 +10,28 @@ import {
     IHighchartsParentTick,
     IHighchartsPointObject,
 } from "../../../../interfaces/DrillEvents";
+import { isBulletChart } from "../../utils/common";
 
-function getDDPointsInParentTick(axis: any, tick: IHighchartsParentTick): IHighchartsPointObject[] {
+export function getDDPointsInParentTick(
+    axis: any,
+    tick: IHighchartsParentTick,
+    chartType: ChartType,
+): IHighchartsPointObject[] {
     const { startAt, leaves } = tick;
     const ddPoints: IHighchartsPointObject[] = []; // drilldown points
 
     for (let i = startAt; i < startAt + leaves; i++) {
         ddPoints.push(...axis.getDDPoints(i));
     }
 
+    if (isBulletChart(chartType)) {
```

Comment:
Isn't it safe to do that all the time no matter what chart is selected?

Reply:
We should not have any other visualization which drilldown events contain target, so isn't it more efficient to do it that way?

Reply:
I more on the side of simpler code than efficiency but I would like to know the opinion of somebody else as I don't have a strong opinion here.

Reply:
If it makes sense to do for all highchart visualizations and it doesn't break them, i would remove that condition. And if you really want efficient code, it's more efficient to put it in that for loop up there than doing it again 😄

Reply:
...and possibly add a comment because it's really not obvious that it makes sense only for bullet chart